components/Cards/docs/accessibility.md

```diff
@@ -0,0 +1,66 @@
+### Accessibility Labels
+
+To help ensure your Cards are accessible to as many users as possible, please be sure to review the following 
+recommendations:
+
+#### Accessibility Labels for Nested Elements
+
+Set an appropriate[`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) 
+for meaningful content that is not textual by default, like images. Also make sure to enable the element's 
+[`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+myImageView.isAccessibilityElement = true
+myImageView.accessibilityLabel = "Image title or some info"
+```
+
+Objective-C
+```objc
+myImageView.isAccessibilityElement = YES;
+myImageView.accessibilityLabel = @"Image title or some info";
+```
+
+#### Accessibility Label for Tappable Cards
+
+Tappable or Selectable Cards are often used as items in a collection. In this mode, Cards often expose a single 
+viewable or actionable item to assistive technologies, by aggregating the cards content, instead of exposing their 
+nested elements individually. This mode works best when there is a limited number of nested elements that can 
+be described as a sentence. To enable this mode, set an appropriate [`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) value 
+for the Card, and make sure to update it each time the Card's content changes. Additionally, make sure to enable 
+the card's [`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement)
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+card.isAccessibilityElement = true
+card.accessibilityLabel = "Location \(locationLabel.accessibilityLabel) has 
```

Comment:
I think an example that doesn't user string concatenation would be better.  Instead, how about a custom label based on the same data that those elements might use instead?  I think we should also avoid plurals as well, since [plurals across languages can be very complex](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LocalizingYourApp/LocalizingYourApp.html#//apple_ref/doc/uid/10000171i-CH5-SW10).

```swift
card.accessibilityLabel = "Location \(userLocation.name) is popular with users who enjoy \(userLocation.popularActivityMatchingUserProfile(userProfile)"
```